packages/gatsby/src/utils/__tests__/handle-flags.ts

```diff
@@ -441,8 +439,120 @@ describe(`handle flags`, () => {
         - FAST_DEV · Enable all experiments aimed at improving develop server start time
         - ALL_COMMANDS · (Umbrella Issue (test)) · test
         - YET_ANOTHER · (Umbrella Issue (test)) · test
-        - PARTIAL_RELEASE · (Umbrella Issue (test)) · test
-        - PARTIAL_RELEASE_ONLY_NEW_LODASH · (Umbrella Issue (test)) · test
```

Comment:
case of number not matching - we were printing `There are 3 other flags` and then list 5 flags